src/main/java/seedu/StudentListCollection.java

```diff
@@ -0,0 +1,21 @@
+package seedu;
+
+import java.util.ArrayList;
+
+public class StudentListCollection extends ArrayList<StudentList> {
+
+    public ArrayList<StudentList> studentListCollection;
+
+    public StudentListCollection() {
+        studentListCollection = new ArrayList<>();
+    }
+
+    @Override
+    public String toString() {
+        String data = "";
+        for (StudentList studentList: this.studentListCollection) {
+            data = data.concat("\n" + studentList.toString());
```

Comment:
Can try to have `studentList.toString() + "\n"`?

I used `System.lineSeparator()` instead of `\n`, maybe can help change that?